src/test/token/ERC20MultiVotes.t.sol

```diff
@@ -77,6 +77,14 @@ contract ERC20MultiVotesTest is DSTestPlus {
         }
     }
 
+    function testDelegateToAddressZero() public {
+        token.mint(address(this), 100e18);
+        token.setMaxDelegates(2);
+
+        hevm.expectRevert(abi.encodeWithSignature("DelegationError()"));
```

Comment:
Can replace `abi.encodeWithSignature` with `bytes` if helpful/nicer

Reply:
Error encoding is different for custom errors. abi.encodeWithSignature does a bytes4(keccak256()) (like a function selector) because thats the data that gets thrown in a custom error. `bytes` is only for normal solidity string revert errors